PolyDock/PolyDock/src/pd/ecs/sys/tabsHeader/TabsActivationSystem.cpp

```diff
@@ -26,7 +26,7 @@ void TabsActivationSystem::update(entt::registry& registry, entt::entity root) c
 			// @todo(squares): sort entities by depth (or maybe disable hovering anything other than top window)
 			for (auto entity : view)
 			{
-				auto& hovered = view.get<HoveredTabComponent>(entity);
+				auto hovered = view.get<HoveredTabComponent>(entity);
```

Comment:
Why & is removed? I would rather add const than remove & because size of this component can possibly change and after some time when someone will want to modify this code he/she can assume that 'hovered' is a reference not a copy (i always used references or const references to components in the rest of the codebase) and it can lead to an irritating bug :)

Reply:
looks like some mistake, fixed.